submit.daint.slurm

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
-#SBATCH --constraint=gpu
 #SBATCH --job-name=<NAME>
+#SBATCH --output=<OUTFILE>
 #SBATCH --ntasks=<NTASKS>
 #SBATCH --ntasks-per-node=<NTASKSPERNODE>
 #SBATCH --cpus-per-task=<CPUSPERTASK>
-#SBATCH --output=<OUTFILE>
-#SBATCH --time=00:45:00
+#SBATCH --partition=<PARITION>
+#SBATCH --time=00:30:00
```

Comment:
Why change the default time from 45 minutes to 30? Is this to try to stop burning through our allocation so quickly?